easybuild/easyblocks/n/numpy.py

```diff
@@ -214,9 +238,20 @@ def get_libs_for_mkl(varname):
 
     def test_step(self):
         """Run available numpy unit tests, and more."""
-        super(EB_numpy, self).test_step()
 
-        # temporarily install numpy, it doesn't alow to be used straight from the source dir
+        if self.cfg['runtest']:
+            # Let's handle the output from the numpy test suite ourselves
+            testcmd_output, testcmd_exit_code = super(EB_numpy, self).test_step(return_testcmd_output=True)
+
+            numpy_testsuite_summary = parse_numpy_test_suite_output(testcmd_output)
+
+            if 'failed' in numpy_testsuite_summary or 'error' in numpy_testsuite_summary:
```

Comment:
Do we need to be careful here with `failed` vs `xfailed`?

Reply:
`numpy_testsuite_summary` is a dict at this point, I'm searching for a specific key.

Reply:
I guess my question is actually, what does `xfailed` mean? Expected failures?

Reply:
Hard to find but the answer is yes (from looking at https://github.com/scipy/scipy/commit/2c27ec48c249e51bdeeb9ba7a3d96db04cfbb6de)